frontend/src/components/nodes/NodeNav.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from "next/link";
+
+import React from "react";
+import { Row, Col, Spinner } from "react-bootstrap";
+
+import NodesApi from "../../libraries/explorer-wamp/nodes";
+
+interface State {
+  validators?: number;
+  others?: number;
```

Comment:
These are not self-descriptive names.

* `validatorsCount`
* `nonValidatorsCount` is better than `other`